Dockerfile

```diff
@@ -0,0 +1,463 @@
+# syntax = docker/dockerfile:1.2
+
+ARG KUBE_VER=v1.21.0
```

Comment:
One thing to watch out for is duplicating version variables might lead to some divergence. Might be worth considering picking a location and using it as a source of authority... otherwise the risk is to get out of sync.

Reply:
Yes, agree - I've been contemplating moving these to the makefile but left it as future improvement.